README.md

```diff
@@ -28,6 +37,37 @@ Extension maintainers are responsible for their extensions.
 Updates to the extensions will also be reviewed by the steering council.
 The steering council reserves the right to reassign extensions to other maintainers in case of prolonged inactivity or other reasons at its own discretion.
 
+## Registering a registered attribute
+
+Strictly speaking, registered attributes are not extensions, because the `attributes` dictionary in Zarr arrays and groups may be populated with arbitrary metadata.
+Therefore, implementations do not have strict guarantees about the contents of the `attributes` dictionary and are not required to fail if the `attributes` dictionary contains unknown keys.
+
+However, there are a number of attribute keys that are commonly used within Zarr community as conventions for domain-specific metadata.
+Therefore, this repository provides a catalog of registered attributes for coordination and discovery purposes.
+
+The process for registering an attribute is similar to registering an extension.
+Open a new PR with a new directory under the `attributes` directory, with the top-level key of the registered attribute.
+
+ * [Attributes](./attributes/README.md)
+
+Registered attributes MUST have a `README.md` file that briefly describes the attribute.
+It is RECOMMENDED that the `README.md` links to external specification documents with the following contents:
```

Comment:
Is there an intended difference between this sentence and the one for extensions?